text-ui-test/input.txt

```diff
@@ -1,2 +1,2 @@
-James Gosling
+list
```

Comment:
Maybe consider adding more input tests so that wwe can test all the functions of the program.